applications/NeuroZK.md

```diff
@@ -0,0 +1,266 @@
+- https://www.linkedin.com/in/sgid8t2c
+- https://www.linkedin.com/in/kresna-sucandra
+
+## Development Status :open_book:
+
+We have completed the development of an on-chain task verification system under CyberDock as a part of our previous grant application, it contained three parts - Substrate pallets, Front end app and k3s worker configurations. While task verification made more sense for batch files, we also developed log streaming to provide work proofs for continously running applications.
+
+<img width="1448" alt="7" src="https://github.com/user-attachments/assets/67d7535a-8ea6-4660-8bdd-bc8de22ea639">
+
+Through this process, we identified the need for having a much simpler provider onboarding experience, so we are developing a custom node implementation called Cyborg worker nodes using which providers can connect their resources to the Cyborg parachain just by downloading and installing an executable file on their VMs. This also aligns with our focus to launch a plug and play hardware series called 'Cyborg miners' that directly connects to the parachain like an ASIC miner.
+
+Consequently, we will be delivering a ZK based offchain compute verification system, that validates the integrity of executing pre-trained AI models for decentralized AI inferencing on crowdsourced GPUs. For this grant we will use ZK and TEE to uphold data privacy in the system.
+
+We are also working concurrently on integrations with Peaq Network (DID pallet) and [CESS](https://cess.network/) for a decentralized storage product called CyStore. Additionally, we have garnered interest from Peaq to open an HRMP channel for XCM interactions between our chains once we are live.
+
+We have created a UX demo to understand our app flow [Cyborg Network UX demo](https://drive.google.com/file/d/1zrN4TRMDd5ON2xH9F00upcyK-3ofwV4r/view?usp=drivesdk)
+
+## Development Roadmap :nut_and_bolt:
+
+The grant delivery will have 3 distinct parts split into 3 milestones delivered every 2 months,
+
+1. TEE Oracle Implementation:
+
+* Develop an [ORML](https://github.com/open-web3-stack/open-runtime-module-library) based custom oracle for private and secure data exchange
+* A custom pallet logic for handling the oracle
+* UI updates for Oracle
+
+2. Development of Cyborg Worker Node:
+
+* Cyborg Worker Node defintion and configuration to enable compute providers to directly download executable and deploy worker nodes without clustering
+* System Pallets for Cyborg parachain (Payments and Inventory)
+* UI updates for Worker Nodes (Currently marked coming soon)
+
+3. NeuroZK - phase 1:
+
+* A RISC Zero based offchain ZK proof generation module
+* Pallet implementation for proof verification
+* System pallet for parachain (Worker Rewards)
+
+### Overview
+
+- **Total Estimated Duration:** 6 months
+- **Full-Time Equivalent (FTE):** 2 ( 1 Full time Developer at 1 FTE/month and 2 part time developers each at 0.5 FTE/month)
+- **Total Costs:** 30,000 USD
```

Comment:
You are missing the `DOT %` line, indicating the amount of vested DOT you are willing to receive. See https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L127

Reply:
Thanks @semuelle , I have updated the info as per the latest guidelines